.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -0,0 +1,211 @@
+  - type: checkboxes
+    id: miscellaneous-first-checks
+    attributes:
+      label: Miscellaneous first checks
+      description: Please read these carefully and check the appropriate boxes.
+      options:
+        - label: I checked that all ports are open and not blocked by my ISP / hoster.
+          required: true
+        - label: I know that SSL errors are likely the result of a wrong setup on the user side and not caused by DMS itself. I'm confident my setup is correct.
+          required: true
+  - type: input
+    id: affected-components
+    attributes:
+      label: Affected Component(s)
+      description: What is affected by this bug? Please describe it with only a few words here. Detailed description can be given later.
+      placeholder: No debug output is shown.
+    validations:
+      required: true
+  - type: textarea
+    id: when-does-it-occur
+    attributes:
+      label:  When does this occur and what happened?
+      description: Tell us what happened. Use [fenced code blocks](https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks#fenced-code-blocks) when pasting lots of text!
+      placeholder: Although `DMS_DEBUG=1` is set, I see no debug output.
+      render: Markdown
+    validations:
+      required: true
+  - type: textarea
+    id: what-did-you-expect-to-happen
+    attributes:
+      label:  What did you expect to happen?
+      description: Tell us what you expected.
+      placeholder: I expected to see debug messages.
+      render: Markdown
+    validations:
+      required: true
+  - type: textarea
+    id: how-do-we-replicate-this-issue
+    attributes:
+      label: How do we replicate the issue?
+      description: What did you do and how can we replicate this issue?
+      value: |
+        1.
+        2.
+        3.
+        ...
+      render: Markdown
+    validations:
+      required: true
+  - type: input
+    id: mailserver-version
+    attributes:
+      label: DMS version
+      description: With which version did yor encounter this bug? Paste your image tag here.
+      placeholder: v10.1.2
+    validations:
+      required: true
+  - type: dropdown
+    id: main-memory
+    attributes:
+      label: How much RAM is available to DMS explicitly?
+      description: Please provide the amount of memory that DMS can really use. This includes main memory (RAM), but also swap or zram. If you're using Kubernetes, provide the resource limit here if you set one.
+      options:
+        - <= 1GB
+        - <= 2GB
+        - <= 4GB
+        - \> 4GB
+    validations:
+      required: true
+  - type: dropdown
+    id: cpu-count
+    attributes:
+      label: How many CPU cores / threads are available?
+      description: How many CPU cores or threads can DMS use? If you're using Kubernetes, provide the resource limit here if you set one.
```

Comment:
The only options are cores, do we need to account for threads separately or is cores sufficient?

Beyond k8s, you can set resource limits via Docker Compose AFAIK, possibly with `docker run` too, or with systemd cgroup driver on the target slice (daemon or container). "resource limit" is probably universally understood across all those by anyone that does it? (_although with cgroups you limit by cpu time I think, cores or threads would be unrelated pinning them to a process or assigning affinity, something like that, I'm not familiar with k8s limits_)

Reply:
Done. [998cb4b4884b4efcce5930e2c24d7cc902e658aa]